client/packages/android/.gitignore

```diff
@@ -94,4 +94,10 @@ lint/tmp/
 # capacitor-cordova-android-plugins
 
 # Copied web assets
-app/src/main/assets/public
+# commented out, easier to setup and bundle is not packaged, see README.md
+#app/src/main/assets/public
```

Comment:
This is committed now, not packaging bundle in apk (only need to update server, unless some native functionality is added, i'll update strategy for this in versioning kdd)

Reply:
> remove it if it is not needed anymore?

I did remove it at the start, but then i saw a commented out # capacitor-cordova-android-plugins (with a comment), so  thought to add a comment and keep it commented out. 

I fixed comment from `and` to `when`, does that make sense now ? a703b6e52955a16cb88f7d1389a61bafd28012b3

Reply:
ok thanks, so this dir is under version control instead of the default...